saleor/settings.py

```diff
@@ -674,8 +674,7 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 CACHES = {"default": django_cache_url.config()}
 CACHES["default"]["TIMEOUT"] = parse(os.environ.get("CACHE_TIMEOUT", "7 days"))
 
-# Default False because storefront and dashboard don't support expiration of token
-JWT_EXPIRE = get_bool_from_env("JWT_EXPIRE", False)
+JWT_EXPIRE = True
```

Comment:
Shouldn't we still have `get_bool_from_env` here but with the default value set to `True`?

Reply:
I think it should be always enabled. If you want to disable it locally, use `local_settings.py`.

Reply:
I agree with @korycins, the environment variables aren't exactly for such a case like this when you want to override existing settings?

Reply:
Having it disabled is a security issue so I'm not sure it should be even configurable.